network-api/networkapi/wagtailpages/models.py

```diff
@@ -574,7 +610,6 @@ class BlogPageTag(TaggedItemBase):
 
 
 class BlogPage(FoundationMetadataPageMixin, Page):
-    template = 'wagtailpages/blog_page.html'
```

Comment:
this got landed at some point, but isn't necessary for CMS-controlled pages that have a template based on their model name (we do need explicit template variables for our static template pages)